### ISSUE TITLE
Please add support for custom fine-tune model

### ISSUE THREAD

**Issue by ILG2021** [guillaumekln/faster-whisper#248]

**Comment by guillaumekln** [comment]
What do you mean exactly?

Models that are fined tuned with Transformers are already supported and can be converted as described in the README.

**Comment by ILG2021** [comment]
I mean model = WhisperModel(modelSize, device=device, device_index=int(gpu), compute_type=compute_type) the modeSize can direct use fine tune model on huggingface.

**Comment by ILG2021** [comment]
No, I mean I have converted a fine tune whisper model to ctranslate2 and upload to huggingface. I want to use the ctranslate2 model directly.
